man/systemctl.xml

```diff
@@ -1997,6 +2000,16 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--no-warn</option></term>
+
+        <listitem>
+          <para>Don't generate the warning shown by default when using
+          <command>enable</command> or <command>disable</command> on units
+          with no install information.</para>
```

Comment:
`with no` -> `without`
Also, please clarify `install information`, as like `systemctl` warn. I.e., mention the `[Install]` section and expected settings in the section.

Reply:
"expected settings" might be bloated for the info of this option?